core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -101,6 +102,11 @@
 
     private boolean hideInaccessibleColumns;
     private boolean forceSpillingJoin;
+    /**
```

Comment:
It feels like this change should be part of previous commit as I get the impression that the functionality of previous commit potentially causes regression without the change in this commit

Reply:
We'll figure out whether to squash them before merge